scripts/tgames.lic

```diff
@@ -332,8 +334,7 @@ def lootcheck(tier, game_location)
 			)
 			if result =~ /It is worth (.*) seashells./
 				found_seashells = $1.to_i
-				fput "redeem my seashell"
-				sleep 0.1
+				dothistimeout "redeem my seashell", 3, /^You redeem your #{found_seashells} Ebon Gate seashells/

```

Comment:
This looks better and more feasible than previous code and gives more time for completion. Thanks.